elab-dto/src/main/java/com/tolean/elab/dto/dictionary/DictionaryNewDto.java

```diff
@@ -0,0 +1,16 @@
+package com.tolean.elab.dto.dictionary;
```

Comment:
- [ ] zmień na DictionaryItemNewDto, gdy dodajemy nową pozycję do słownika to wystarczy nam tylko pole "name". Zróbmy tak, że nowo dodana pozycja do słownika z defaultu jest aktywna.